src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -33,7 +35,10 @@ export class ItemEditComponent implements OnDestroy {
   }
 
   getCurrentItem(): void {
-    this.itemSubscription = this.itemDataSource.item$.subscribe(item => {
+    this.itemSubscription = this.itemDataSource.state$.pipe(
+      filter<Ready<ItemData> | Fetching | FetchError, Ready<ItemData>>(isReady),
+    ).subscribe(state => {
+      const item = state.data.item;
```

Comment:
And so, if it is loading or there is an error?

Reply:
My bad, it's now done.